specification/communication/data-plane/SipRouting/preview/2022-09-01-preview/communicationservicessiprouting.json

```diff
@@ -0,0 +1,306 @@
+          "type": "string"
+        },
+        "trunks": {
+          "description": "Gets or sets list of SIP trunks for routing calls. Trunks are represented as FQDN.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Domain": {
+      "description": "Represents Domain.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Domain type",
+          "$ref": "#/definitions/DomainType"
+        }
+      }
+    },
+    "DomainPatch": {
+      "description": "Represents Domain.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Domain Type",
+          "$ref": "#/definitions/DomainType"
+        }
+      }
+    },
+    "SipConfiguration": {
+      "description": "Represents a SIP configuration.\r\nWhen a call is being routed the routes are applied in the same order as in the routes list.\r\nA route is matched by its number pattern.\r\nCall is then directed into route's first available trunk, based on the order in the route's trunks list.",
+      "type": "object",
+      "properties": {
+        "domains": {
+          "description": "Validated Domains.",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/Domain"
+          }
+        },
+        "trunks": {
+          "description": "SIP trunks for routing calls.\r\nMap key is trunk's FQDN (1-249 characters).",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/Trunk"
+          }
+        },
+        "routes": {
+          "description": "Trunk routes for routing calls.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkRoute"
+          }
+        }
+      }
+    },
+    "SipConfigurationPatch": {
+      "description": "Represents a SIP configuration patch.",
+      "type": "object",
+      "properties": {
+        "domains": {
+          "description": "Domains that will be used",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/DomainPatch"
+          }
+        },
+        "trunks": {
+          "description": "SIP trunks for routing calls.\r\nMap key is trunk's FQDN (1-249 characters).",
+          "maxLength": 250,
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TrunkPatch"
+          }
+        },
+        "routes": {
+          "description": "Trunk routes for routing calls.",
+          "maxLength": 250,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrunkRoute"
+          }
+        }
+      }
+    },
+    "DomainType": {
+      "type": "string",
+      "description": "Type of domain",
+      "enum": [
+        "Custom",
```

Comment:
>C [](http://example.com/codeflow?start=9&length=1)

the casing might be result of your implementation.
Do you want to document azure and custom or really Azure and Custom?

Reply:
Changed casing to lower. Thx